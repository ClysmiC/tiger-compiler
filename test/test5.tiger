let
        var a, b : int := 0;
in
		a := 7;
		b := 5;

        if(a % b) then
        	a := a - 10;

        	if(a < b) then
                a := b;
            else
	            if(a > b) then
	                b := a;
	            else
	                a := 0;
	                b := 0;
	            endif;
            endif;
        endif;

        a := b+2;
        a := (5 * 3 - (2 / 3));
        a := 0;
end
