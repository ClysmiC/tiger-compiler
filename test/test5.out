LET VAR ID COMMA ID COLON ID ASSIGN INTLIT SEMI IN IF LPAREN ID EQ ID RPAREN THEN ID ASSIGN ID
Scanner error (line 5): a := b %<---
                        "%" does not begin a valid token.
INTLIT SEMI ENDIF SEMI ID LPAREN ID RPAREN SEMI END
unsuccessful parse
